config/logstash.yml

```diff
@@ -216,6 +216,23 @@
 # Default is false
 # pipeline.separate_logs: false
 #
+# Enables internal collector to shipping directly to Elasticsearch
```

Comment:
Maybe change this to something like 'Enable monitoring via internal collector to an Elasticsearch monitoring cluster`